### PR TITLE
[circle2circle-dredd-recipe-test] Add Net_Mul_Div_001

### PR DESCRIPTION
**Pull request by seanshpark**
This will add Net_Mul_Div_001 to circle2circle-dredd-recipe-test.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #12841 , from draft #12842

File: compiler/circle2circle-dredd-recipe-test/test.lst [hunk 1]
```diff
@@ -56,6 +56,7 @@ Add(Net_InstanceNorm_006 PASS fuse_instnorm)
 Add(Net_InstanceNorm_007 PASS fuse_instnorm)
 Add(Net_Maximum_Minimum_000 PASS transform_min_max_to_relu6)
 Add(Net_Mul_Div_000 PASS fuse_mul_with_div)
+Add(Net_Mul_Div_001 PASS fuse_mul_with_div)
```
Comment (seanshpark) [inline compiler/circle2circle-dredd-recipe-test/test.lst:59]: will sort in next PR

Comment (jinevening) [reply compiler/circle2circle-dredd-recipe-test/test.lst:59]: LGTM

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**